core/src/config/logger.ts

```diff
@@ -1,15 +1,34 @@
 import * as winston from "winston";
+import { ActionheroConfigInterface } from "actionhero";
 
-// learn more about winston v3 loggers @
-// - https://github.com/winstonjs/winston
-// - https://github.com/winstonjs/winston/blob/master/docs/transports.md
+const namespace = "logger";
+
+declare module "actionhero" {
+  export interface ActionheroConfigInterface {
+    [namespace]: ReturnType<typeof DEFAULT[typeof namespace]>;
+  }
+}
+
+/*
+The loggers defined here will eventually be available via `import { loggers } from "actionhero"`
+
+You may want to customize how Actionhero sets the log level.  By default, you can use `process.env.LOG_LEVEL` to change each logger's level (default: 'info')
+
+learn more about winston v3 loggers @
+ - https://github.com/winstonjs/winston
+ - https://github.com/winstonjs/winston/blob/master/docs/transports.md
+*/
+
+type ActionheroConfigLoggerBuilderArray = Array<
+  (config: any) => winston.Logger
+>;
 
 export const DEFAULT = {
-  logger: (config) => {
-    const loggers = [];
-    loggers.push(buildConsoleLogger(process.env.GROUPAROO_LOG_LEVEL));
-    config.general.paths.log.forEach((p) => {
-      loggers.push(buildFileLogger(p));
+  [namespace]: (config: ActionheroConfigInterface) => {
+    const loggers: ActionheroConfigLoggerBuilderArray = [];
+    loggers.push(buildConsoleLogger(process.env.LOG_LEVEL));
```

Comment:
Not sure if this was intentional, but this is changing the log level env var from `GROUPAROO_LOG_LEVEL` to `LOG_LEVEL`, which may have an impact in other places. I would suggest keeping it as `GROUPAROO_LOG_LEVEL` if possible.

Reply:
Good catch!